### DISCUSSION TITLE
48 vulnerabilities from jackson-databind

### DISCUSSION THREAD

**Discussion by AbstractAlao** [graphql-java-kickstart/graphql-java-servlet#462] [Q&A] [answered]
Are there any plans to update the dependencies from jackson-databind?  Seems the latest doesn't have the CVE's

https://mvnrepository.com/artifact/com.graphql-java/graphql-java-servlet/6.1.3

**Comment by oliemansm** [comment, marked as answer]
You're using a very outdated version of graphql-servlet. The group has changed years ago. See: https://mvnrepository.com/artifact/com.graphql-java-kickstart/graphql-java-servlet/12.0.0
